test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1

```diff
@@ -0,0 +1,501 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Function Install-ModuleIfMissing
+{
+    param(
+        [parameter(Mandatory)]
+        [String]
+        $Name,
+        [version]
+        $MinimumVersion,
+        [switch]
+        $SkipPublisherCheck,
+        [switch]
+        $Force
+    )
+
+    $module = Get-Module -Name $Name -ListAvailable -ErrorAction Ignore | Sort-Object -Property Version -Descending | Select-Object -First 1
+
+    if(!$module -or $module.Version -lt $MinimumVersion)
+    {
+        Write-Verbose "Installing module '$Name' ..." -Verbose
+        Install-Module -Name $Name -Force -SkipPublisherCheck:$SkipPublisherCheck.IsPresent
+    }
+}
+
+Function Test-IsInvokeDscResourceEnable
+{
+    return [ExperimentalFeature]::IsEnabled("PSDesiredStateConfiguration.InvokeDscResource")
+}
+
+Describe "Test PSDesiredStateConfiguration" -tags CI {
+    Context "Module loading" {
+        BeforeAll {
+            $commands = Get-Command -Module PSDesiredStateConfiguration
+            $expectedCommandCount = 3
+            if (Test-IsInvokeDscResourceEnable)
```

Comment:
For consistency, open braces in PS script should always be at the end of the line rather than a new line.  Other  lines below have same issue.

Reply:
Code from other repo